vividus-plugin-parquet/build.gradle

```diff
@@ -0,0 +1,16 @@
+project.description = 'Vividus plugin for Parquet'
+ext.sonarProjectName = 'Vividus Plugin - Parquet'
+
+dependencies {
+    api project(':vividus-bdd-engine')
+    implementation project(':vividus-util')
+    implementation project(':vividus-plugin-csv')
+
+    implementation(group: 'org.apache.parquet', name: 'parquet-avro', version: '1.10.1')
+    implementation(group: 'org.apache.hadoop', name: 'hadoop-common', version: '3.2.0')
+    implementation(group: 'org.apache.hadoop', name: 'hadoop-mapreduce-client-core', version: '0.23.1')
```

Comment:
1) where is it used?
2) why not to use version `3.2.0`?

Reply:
Workaround for failed unit tests with
`java.lang.NoClassDefFoundError: org/apache/hadoop/mapreduce/lib/input/FileInputFormat`

Reply:
so it should be `testImplementation`